lib/firebase_options.dart

```diff
@@ -63,7 +63,7 @@ class DefaultFirebaseOptions {
 
   static const FirebaseOptions macos = FirebaseOptions(
     apiKey: 'AIzaSyD648P_AQMaTgT7Pbry-s_jAskmBpABBc0',
-    appId: '1:1080315079875:ios:43e36ef00e1b40e82c8676',
+    appId: '1:1080315079875:ios:3398c66d2dc3c3722c8676',
```

Comment:
@shubhamsinghmutualmobile did you make these changes or the CLI ?

Reply:
CLI

Reply:
A ghost must have updated these then 🤷🏼‍♂️  😆